src/main/java/spoon/Launcher.java

```diff
@@ -371,6 +376,15 @@ protected static JSAP defineArgs() {
 			sw1.setDefault("false");
 			jsap.registerParameter(sw1);
 
+			// Generate only java files specified.
+			opt2 = new FlaggedOption("generate-files");
+			opt2.setShortFlag('f');
+			opt2.setLongFlag("generate-files");
+			opt2.setHelp("Generate only java files specified by their fully qualified name and separated by ':'. This argument only works when your output-type is specified at classes.");
```

Comment:
Only generate the given fully qualified java classes  (separated by ':' if multiple are given)